### DISCUSSION TITLE
Launching animal-ai with Configuration.yaml without python

### DISCUSSION THREAD

**Discussion by mr-cubus** [Kinds-of-Intelligence-CFI/animal-ai#46] [Q&A] [answered]
Hi, is there a way to launch AnimalAI with a specific configuration.yaml without using python? The unity project mentions a `--receiveConfiguration` flag, but I'm having trouble getting it to work on macOS.

Here are the commands I've tried:
```
open MACOS.app --args --receiveConfiguration "animal-ai/configs/competition/02-25-02.yaml"  
MACOS.app/Contents/MacOS/AnimalAI --receiveConfiguration "animal-ai/configs/competition/02-25-02.yaml"
open MACOS.app --args play animal-ai/configs/competition/02-25-02.yaml
MACOS.app/Contents/MacOS/AnimalAI play animal-ai/configs/competition/02-25-02.yaml
```
I've also looked into the GettingStarted guide, which mentions a file named Launching-Animal-AI.md, but I couldn't find it in the repository. Everything works as expected when using the Python wrapper.

Thank you!

**Comment by alhasacademy96** [comment, marked as answer]
Hi there,

The Launching AI.md is still in the works and apologies for the delay. It will contain all you need information regarding launching AAI for training and play modes.

Regarding your question, the `receiveConfiguration` flag is a check for checking the side channels (the connection between python and unity) for a valid connection. It is not a logic for enabling configuration loading natively via unity. This is a feature yet to be implemented. Please see our Roadmap(.md) doc for more details. You are free to contribute if you wish to but as it stands this is not going to be implemented in the next few months. 

I hope i helped even a bit :)

Ibrahim
Lead Software Developer

**Reply by mr-cubus** [reply]
Thank you! :)

**Reply by alhasacademy96** [reply]
Launching AAI.md is now complete :) @mr-cubus Have a look!
